frontend/src/components/Dropdown.vue

```diff
@@ -0,0 +1,157 @@
+<script setup lang="ts">
+import { debugPort } from 'process';
+import { ref } from 'vue'
+
+const currentDataset = ref(0)
+
+const dropdownVisible = ref(false)
+
+const options = [
+  {
+    name: "STEM degrees conferred by level",
+    source: "IPEDS",
+    variables: ["Year", "Sex", "Race", "Degree Level", "Disability status"],
+    year: "2001 - 2021"
+  },
+  {
+    name: "National survey of college graduates salaries",
```

Comment:
Super nit: At some point we'll want to unify the capitalization standards and source abbreviations. Same with fact checking the year ranges offered.
Doesn't need to be in this PR but want to flag.